hammer_cli_reference/playbook.yaml

```diff
@@ -0,0 +1,100 @@
+---
+- name: "Check prerequisites on developer-system"
+  connection: local
+  hosts: developer-system
+  vars:
+    repo_path: "/home/user"
+    repo_remote: "git@github.com:theforeman/foreman-documentation.git"
+
+  tasks:
+    - name: "Verify that user input is overwritten"
+      ansible.builtin.assert:
+        that: repo_path != "/home/user"
+
+    - name: "Verify that shellscript is present"
+      ansible.builtin.stat:
+        path: "{{ repo_path }}/guides/scripts/generate-hammer-reference.sh"
+      register: result
+
+    - name: "End playbook if shellscript is not present"
+      ansible.builtin.fail:
+        msg: "Shellscript to convert markdown to asciidoc is not present or executable"
+      when:
+        - not result.stat.executable
+        - not result.stat.exists
+
+    - name: "Delete markdown file if present"
+      ansible.builtin.file:
+        path: "{{ repo_path }}/guides/hammer_full_help.md"
+        state: absent
+
+    - name: "Verify that foreman-documentation is present"
+      ansible.builtin.git:
+        clone: false
+        dest: "{{ repo_path }}"
+        repo: "{{ repo_remote }}"
+        update: false  # noqa: latest
+        version: "HEAD"
+
+- name: "Generate Hammer CLI help"
+  hosts: foreman-katello
+
+  tasks:
+    - name: "Update DNF cache"
+      ansible.builtin.dnf:
+        update_cache: true
+
+    - name: "Install all available Hammer CLI packages"
+      ansible.builtin.package:
+        name: "rubygem-hammer_cli*"
```

Comment:
In my tests I had to install plugins as well, not only their cli-packages. I did not completely test which plugins were required but eventually just enabled all plugins.

This goes into the same direction as @Lennonka 's comment  here and also the next one by @Lennonka about `discovered_hosts` can be solved by installing all plugins.

Reply:
I have implemented a hack-ish way to do that.